modules/dream2nix/rust-crane/devshell.nix

```diff
@@ -76,6 +78,7 @@
     // {
       inherit name;
       passthru.env = _shellEnv;
+      nativeBuildInputs = _shellEnv.nativeBuildInputs ++ [cargo];
     };
 in
-  (mkShell.override {stdenv = (l.head drvs).stdenv;}) shellEnv
+  (mkShell.override {stdenv = (l.head drvs).config.mkDerivation.stdenv or stdenv;}) shellEnv
```

Comment:
I think the option `config.mkDerivation.stdenv` does not exist.

Reply:
done, i am pulling it from `mainDrv.out.stdenv` now instead, i missed that when i first wrote that i think